app/src/main/res/layout/project_metadata_view.xml

```diff
@@ -45,6 +40,7 @@
       style="@style/BodyPrimaryMedium"
```

Comment:
ooo looks like we bumped down the font size, so the metadata text views in this file can be `CaptionPrimaryMedium`